specification/resources/resource-manager/readme.md

```diff
@@ -101,6 +101,8 @@ input-file:
 - Microsoft.Authorization/stable/2021-06-01/policyDefinitions.json
 - Microsoft.Authorization/stable/2021-06-01/policySetDefinitions.json
 - Microsoft.Authorization/preview/2020-07-01-preview/policyExemptions.json
+- Microsoft.Authorization/preview/2022-08-01-preview/policyVariables.json
+- Microsoft.Authorization/preview/2022-08-01-preview/policyVariableValues.json
```

Comment:
@leni-msft I think addition of policyVariableValues is breaking the sdk schema generation. I've sent an email to sdkowners@microsoft.com about this in detail. Can you help me get some traction to see if this is a bug, or something I'm doing wrong. Thx. Ping me on teams or here when you get a chance please.

Reply:
@jorgecotillo could you help check the error during schema generation `An item with the same key has already been added. Key: variables_values_childResource` ?

Reply:
I sent you both a detailed email. I can repro this locally which helps with the iteration speed and the errors.

Reply:
it's been fixed, so resolving this.